spec/haskell/src/SEL4/Object/ObjectType.lhs

```diff
@@ -519,5 +520,3 @@ The following two functions returns the base and size of the object a capability
 >     = 1 -- error in haskell
 > capUntypedSize (IRQHandlerCap {})
 >     = 1 -- error in haskell
-
-
```

Comment:
does this not trigger "no empty line at end of file" lint?

Reply:
I'm confused. The lint wants me to not have an empty line at the end (a newline, yes, but no empty line).

Reply:
I think I'm the one confused. In jedit, you see an empty line at the end, in vim you don't, and in git diff I'm not sure anymore